Dockerfile

```diff
@@ -36,25 +64,32 @@ RUN pip3 install unittest-xml-reporting
 # The downloaded script is needed since git-lfs is not available per default for Ubuntu 16.04
 RUN curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash && sudo apt-get install -y git-lfs
 
-
-# Remove all apt lists to avoid build caching
-RUN rm -rf /var/lib/apt/lists/*
-
 # en_US.utf8 is required by Yocto sanity check
+RUN /usr/sbin/locale-gen en_US.UTF-8
 RUN localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
 RUN echo 'export LC_ALL="en_US.UTF-8"' >> /etc/profile
 ENV LANG en_US.utf8
 
+RUN useradd -U -m yoctouser
+
+# Enables us to overwrite the user and group ID for the yoctouser. See below
+ARG userid=1000
+ARG groupid=1000
```

Comment:
Why did you move this? (I guess intent of original author was probably to have ARG at top. Do not know for sure nor what best practice. Regardless, unnecessary to move them if not needed. If it is a cleanup it should be a separate commit.)